docs/source-pytorch/governance.rst

```diff
@@ -57,15 +57,6 @@ Alumni
 - Peter Yu (`yukw777 <https://github.com/yukw777>`_)
 - Roger Shieh (`s-rog <https://github.com/s-rog>`_)
 
-
-Releases
```

Comment:
"Releases" and "API Evolution" sections are moved to a new standalone versioning policy page with improvements.